WordPress/gutenberg
# Tools: Upgrade Lerna to the latest version

## What?

From https://blog.nrwl.io/lerna-is-dead-long-live-lerna-61259f97dbd9:

> We, [Nrwl](https://nrwl.io/), the company behind [Nx](https://nx.dev/), are taking over the stewardship of Lerna.js, the popular JS monorepo tool. [Here’s the official announcement on the Lerna repo](https://github.com/lerna/lerna/issues/3121). We are thrilled and committed to helping the Lerna community move forward!

Lerna had a few new releases, including:

https://blog.nrwl.io/lerna-5-1-new-website-new-guides-new-lerna-example-repo-distributed-caching-support-and-speed-64d66410bec7

It's now on version 5.5.x, but the functionality we use is mostly based around npm publishing and it should remain the same. The idea behind updating it to the latest version is related to ensuring we install more recent version of Lerna's dependencies to avoid security issues in the future.

## Testing Instructions





I can run npm publishing from `trunk` with the `next` dist-tag if that helps, but I hoped we could run it from GitHub UI after landing this PR.

I tested locally with:

```bash
npm run publish:check
```

and it correctly reported 86 modified packages.
